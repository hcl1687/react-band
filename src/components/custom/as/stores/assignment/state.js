export default (RB_CONTEXT) => {
  return {
    items: [],
    total: 0
  }
}
