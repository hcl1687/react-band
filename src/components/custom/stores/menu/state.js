export default {
  MENU_STATUS: true,
  MENU_ENABLED: true
}
