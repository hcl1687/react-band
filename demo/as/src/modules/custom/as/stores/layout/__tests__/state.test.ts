import state from '../state'

describe('custom/as/stores/layout/state', () => {
  it('should render correctly', async () => {
    expect(state).toMatchSnapshot()
  })
})
