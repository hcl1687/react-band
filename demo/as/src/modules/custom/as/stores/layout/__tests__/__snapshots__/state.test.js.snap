// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom/as/stores/layout/state should render correctly 1`] = `
Object {
  "HEAD_STATUS": true,
  "LAYOUT_MODE": "left",
  "LEFT_STATUS": true,
}
`;
